actionpack/lib/action_dispatch/middleware/debug_view.rb

```diff
@@ -56,5 +56,13 @@ def render(*)
     def protect_against_forgery?
       false
     end
+
+    def params_readable?
```

Comment:
`params_readable?` to me reads like "can be read" rather than "is not malformed". I'm struggling to come up with a better name though.